src/System.ServiceModel.NetTcp/ref/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "System.ObjectModel": "4.0.0",
-    "System.Runtime": "4.0.0",
+    "System.Runtime": "4.0.20",
```

Comment:
This change just fixed a conflict warning.
This project references System.ServiceModel.Primitives version 4.1.0 which requires System.Runtime version 4.0.20 this created the conflict.

The other Façade ref project.json files reference System.ServiceModel.Primitives version 4.0.0 therefore no conflicts for them.